src/renderers/shaders/ShaderLib.js

```diff
@@ -82,12 +82,15 @@ var ShaderLib = {
 			UniformsLib.displacementmap,
 			UniformsLib.roughnessmap,
 			UniformsLib.metalnessmap,
+			UniformsLib.glossinessmap,
 			UniformsLib.fog,
 			UniformsLib.lights,
 			{
 				emissive: { value: new Color( 0x000000 ) },
 				roughness: { value: 0.5 },
 				metalness: { value: 0 },
```

Comment:
That isn't what I edited.
I already noticed that and sent another PR.

https://github.com/mrdoob/three.js/pull/11039